src/hooks/tasksReducer.js

```diff
@@ -0,0 +1,22 @@
+const tasksInitialState = [];
+
+const actionTypes = {
+  ADD_TASK: 'ADD_TASK',
+  REMOVE_TASK: 'REMOVE_TASK',
+  UPDATE_TASK: 'UPDATE_TASK',
+};
```

Comment:
The action-type object is used to just avoid some errors that may arise due to spelling mistakes in action types?

Reply:
One scenario is spelling mistakes, the other is if you want to change the value of any of them then it ca be done in just one place